main.tf

```diff
@@ -79,7 +71,7 @@ data "aws_kms_key" "lambda" {
 module "default_label" {
   enabled = var.enabled
 
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
+  source = "../../../../../modules/terraform-null-label"
```

Comment:
Where is this `modules` directory?

Reply:
In LMS Terraform repo 
reference link:
[[Pull Request 15180](https://dev.azure.com/GCENextGen/NextGenLMS/_git/terraform/pullrequest/15180): changed the provider version to 4.52.0](url)